pkg/render/guardian.go

```diff
@@ -127,9 +127,12 @@ func (c *GuardianComponent) Objects() ([]client.Object, []client.Object) {
 		c.cfg.TrustedCertBundle.ConfigMap(GuardianNamespace),
 		// Add tigera-manager service account for impersonation
 		CreateNamespace(ManagerNamespace, c.cfg.Installation.KubernetesProvider, PSSRestricted),
-		managerServiceAccount(),
+		managerServiceAccount("tigera-manager"), // TODO
```

Comment:
This actually can probably be left as it is. Voltron will need to translate serviceaccount names.